bundle/permissions/permission_diagnostics.go

```diff
@@ -0,0 +1,179 @@
+package permissions
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/log"
+)
+
+const CheckPermissionsFilename = "permissions.check"
+
+type reportPermissionErrors struct{}
+
+func PermissionDiagnostics() bundle.Mutator {
+	return &reportPermissionErrors{}
+}
+
+func (m *reportPermissionErrors) Name() string {
+	return "CheckPermissions"
+}
```

Comment:
Most of the other code uses the same name for the filename, the constructor, the type, and the name.

Reply:
Please consolidate the naming.